src/main/java/com/gregtechceu/gtceu/api/gui/widget/ColorBlockWidget.java

```diff
@@ -47,8 +58,8 @@ public void drawInBackground(@NotNull GuiGraphics graphics, int mouseX, int mous
             currentColor = colorSupplier.getAsInt();
         }
         final int BORDER_COLOR = 0xFF000000;
-
-        graphics.fill(x, y, x + width, y + height, currentColor);
+        int opaqueColor = isShowAlpha ? currentColor : (currentColor & 0x00FFFFFF) | 0xFF000000;
```

Comment:
you don't need to remove the alpha before readding it.
```suggestion
        int opaqueColor = isShowAlpha ? currentColor : currentColor | 0xFF000000;
```